toketmaster/tests/test_tasa.py

```diff
@@ -0,0 +1,86 @@
+import time
+import unittest
+
+from algosdk.error import IndexerHTTPError
+
+from toketmaster.src import client, models, tasa
+
+
+def wait_to_complete(func, *args, **kwargs):
```

Comment:
Move this to a helper class, like https://github.com/algocertified/sandbox/blob/master/toketmaster/algorand_test_helper.py

Reply:
Done: https://github.com/algocertified/sandbox/pull/2/commits/9626629b65a0ed0515797221b632426e1495d021